Sming/SmingCore/Network/Http/HttpServerConnection.cpp

```diff
@@ -280,6 +280,8 @@ int HttpServerConnection::staticOnBody(http_parser* parser, const char* at, size
 	}
 
 	// TODO: ...
+	connection->request.setBody((const uint8_t*)&at, length);
```

Comment:
`connection->request.setBody((const uint8_t*)&at, length);`

That is something that we don't do. Because it is the easiest way to crash remotely the http server and the device. Just send a request with very long body. What we do is use body parsers and decide in there how to operate over the incoming response body.

That said, please, remove that new line.